uber/templates/accounts/attendees.html

```diff
@@ -0,0 +1,49 @@
+{% if attendees == c.ROW_LOAD_LIMIT %}
```

Comment:
I think you meant to apply a ``|length`` here because as-is you're comparing the actual list of attendees to the count.  (It looks like this was already a bug which only shows up as new because the code was moved, but we should make sure to fix it here.)